packages/ember-htmlbars/tests/helpers/get_test.js

```diff
@@ -1,495 +0,0 @@
-});
-
-QUnit.test('should handle object values as nulls', function() {
-  var context = {
-    colors: null
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get colors \'apple\'}}] [{{if true (get colors \'apple\')}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[] []', 'should return \'\' for {{get colors \'apple\'}} (colors = null)');
-
-  run(function() {
-    view.set('context.colors', { apple: 'green', banana: 'purple' });
-  });
-
-  equal(view.$().text(), '[green] [green]', 'should return \'green\' for {{get colors \'apple\'}} (colors = { apple: \'green\', banana: \'purple\' })');
-
-  run(function() {
-    view.set('context.colors', null);
-  });
-
-  equal(view.$().text(), '[] []', 'should return \'\' for {{get colors \'apple\'}} (colors = null)');
-});
-
-QUnit.test('should handle object keys as nulls', function() {
-  var context = {
-    colors: { apple: 'red', banana: 'yellow' },
-    key: null
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get colors key}}] [{{if true (get colors key)}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[] []', 'should return \'\' for {{get colors key}}  (key = null)');
-
-  run(function() {
-    view.set('context.key', 'banana');
-  });
-
-  equal(view.$().text(), '[yellow] [yellow]', 'should return \'yellow\' for {{get colors key}} (key = \'banana\')');
-
-  run(function() {
-    view.set('context.key', null);
-  });
-
-  equal(view.$().text(), '[] []', 'should return \'\' for {{get colors key}}  (key = null)');
-});
-
-QUnit.test('should handle object values and keys as nulls', function() {
-  var context = {
-    colors: null,
-    key: null
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get colors key}}] [{{if true (get colors key)}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[] []', 'should return \'\' for {{get colors key}}  (colors=null, key = null)');
-});
-
-QUnit.test('get helper value should be updatable using {{input}} and (mut) - dynamic key', function() {
-  var context = {
-    source: EmberObject.create({
-      banana: 'banana'
-    }),
-    key: 'banana'
-  };
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    context: context,
-    template: compile(`{{input type='text' value=(mut (get source key)) id='get-input'}}`)
-  });
-
-  runAppend(view);
-
-  equal(view.$('#get-input').val(), 'banana');
-
-  run(function() {
-    view.set('context.source.banana', 'yellow');
-  });
-
-  equal(view.$('#get-input').val(), 'yellow');
-
-  run(function() {
-    view.$('#get-input').val('some value');
-    view.childViews[0]._elementValueDidChange();
-  });
-
-  equal(view.$('#get-input').val(), 'some value');
-  equal(view.get('context.source.banana'), 'some value');
-});
-
-QUnit.test('get helper value should be updatable using {{input}} and (mut) - dynamic nested key', function() {
```

Comment:
:heavy_check_mark: get helper value should be updatable using {{input}} and (mut) - dynamic nested key